tweet_on_upcoming_and_recently_released_movies_using_azure_openai/main.bal

```diff
@@ -0,0 +1,66 @@
+import ballerina/io;
+import ballerinax/azure.openai.text;
+import ballerinax/themoviedb;
+import ballerinax/twitter;
+
+configurable string moviedbApiKey = ?;
+
+configurable string openAIToken = ?;
+configurable string serviceUrl = ?;
+configurable string deploymentId = ?;
+
+configurable string twitterApiKey = ?;
+configurable string twitterApiSecret = ?;
+configurable string twitterAccessToken = ?;
+configurable string twitterAccessTokenSecret = ?;
+
+const API_VERSION = "2023-03-15-preview";
+const MAX_TWEET_LENGTH = 280;
+const MAX_TOKENS = 1000;
+const NO_OF_MOVIES = 5;
+
+// Twitter API client configuration
+twitter:ConnectionConfig twitterConfig = {
+    apiKey: twitterApiKey,
+    apiSecret: twitterApiSecret,
+    accessToken: twitterAccessToken,
+    accessTokenSecret: twitterAccessTokenSecret
+};
+
+public function main() returns error? {
+    // Get information on upcoming and recently released movies from TMDB
+    final themoviedb:Client moviedb = check new themoviedb:Client({apiKey : moviedbApiKey});
+    themoviedb:InlineResponse2001 moviedbRes = check moviedb->getUpcomingMovies();
+
+    // Generate a creative tweet using Azure OpenAI   
+    final text:Client azureOpenAI = check new (
+        config = {auth: {apiKey: openAIToken}},
+        serviceUrl = serviceUrl
+    );
+
+    string prompt = "Instruction: Generate a creative and short tweet below 250 characters about the following upcoming and recently released movies. Movies: ";
+    foreach int i in 1 ... NO_OF_MOVIES {
+        var movie = moviedbRes.results[i - 1];
+        prompt += string `${i.toString()}. ${movie.title} `;  
+    }
+
+    text:Deploymentid_completions_body completionsBody = {
+        prompt,
+        max_tokens: MAX_TOKENS
+    };
+    text:Inline_response_200 completion = check azureOpenAI->/deployments/[deploymentId]/completions.post(API_VERSION, completionsBody);
```

Comment:
Break the line at max of 120 chars for readability.

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/63/commits/f7ce252feadcc1c2f7f19f7b18a3f078cbe22927.